Tests/SentryTests/SentryInterfacesTests.m

```diff
@@ -88,7 +88,6 @@ - (void)testEvent
         @"level" : @"info",
         @"environment" : @"bla",
         @"platform" : @"cocoa",
-        @"sdk" : @ { @"name" : @"sentry.cocoa", @"version" : SentryMeta.versionString },
```

Comment:
`h`: Instead of removing this here can we please add `event.sdk = @ { @"name" : @"sentry.cocoa", @"version" : SentryMeta.versionString };` above to make sure we test the serialization of the sdk property?